gpu_bdb/benchmark_runner.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+USERNAME=$(whoami)
+
+if [ -z "$GPU_BDB_HOME" ]
+then
+    GPU_BDB_HOME=/raid/$USERNAME/prod/gpu-bdb
+else
+    GPU_BDB_HOME=$GPU_BDB_HOME
+fi
+
+INCLUDE_DASK=true
+INCLUDE_BLAZING=false
+N_REPEATS=1
+
+# Dask queries
+if $INCLUDE_DASK; then
+    for qnum in {01..30}
+    do
+        cd $GPU_BDB_HOME/gpu_bdb/queries/q$qnum/
+        for j in $(seq 1 $N_REPEATS)
+        do
+            python gpu_bdb_query_$qnum.py --config_file ../../benchmark_runner/benchmark_config.yaml
+            sleep 3
+        done
+        sleep 3
+    done
+fi
+
+# BlazingSQL Queries
+if $INCLUDE_BLAZING; then
+    for qnum in {01..30}
+    do
+        cd $GPU_BDB_HOME/gpu_bdb/queries/q$qnum/
+        for j in $(seq 1 $N_REPEATS)
+        do
+            python gpu_bdb_query_$qnum\_sql.py --config_file ../../benchmark_runner/benchmark_config.yaml
```

Comment:
Just noting that launching a process for each script will add library import-related overheads which for 30 queries quickly add up for the whole benchmark. 

IIRC they add to like `30-50s` of the whole run time which is significant

Reply:
This is a good point, and something I had forgotten. Let's discuss this more offline